invision-app/app/components/noMoviesPage/noMoviesPage.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { ErrorBoundary } from 'react-error-boundary';
+import { AddMovieButton } from '../addMovieButton'
+import { MovieFiterAndSort } from '../movieFiterAndSort';
+
+import { Header } from '../Header';
+import { Footer } from '../Footer';
+import './noMoviesPage.scss';
+
+export function NoMoviesPage(
+    {
+        searchState,
+        setSearchState,
+        fetchMovies,
+        setOrderState,
+        setGenreState
+    }
+) {
+    // const [isDetailsPage, setDetialsPage] = useState(false);
+    // const displayMainPage = useCallback(() => {
+    //     setDetialsPage(false);
+    //   }); 
+    //   const showMovieDetails = useCallback(() => {
+    //     setDetialsPage(true);
+    //   });
+
+    return (
+        <>
+            <Header></Header>
+            <main className='no-movies-container'>
+                <AddMovieButton searchState={searchState} setSearchState={setSearchState} fetchMovies={fetchMovies} history={history} />
+
+                <MovieFiterAndSort setOrderState={setOrderState} setGenreState={setGenreState} />
+
+                <div className='no-movies-content'>
```

Comment:
You already have **Home** component with the same structure. Moreover, you should display **No Movie Found** message when search returns empty list and at first loading. Looks like you should update **MovieList** component to display this message